packages/opentelemetry-plugin-http/src/http.ts

```diff
@@ -407,7 +410,12 @@ export class HttpPlugin extends BasePlugin<Http> {
       const span = plugin._startHttpSpan(operationName, spanOptions);
       plugin._tracer
         .getHttpTextFormat()
```

Comment:
As per my understanding, we should remove `getHttpTextFormat` and `getBinaryFormat` from tracer API and SDK (i.e. `tracer` instance should not be aware of propagators). `ContextPropagators` (or similar) should leave as its own (similar to `globalMeterProvider`) with set and get (defaulting to `HttpTraceContext`) methods and plugins should access it directly in order to inject and extract the data.

Reply:
Yes this is true. I wanted to make the multi/global propagators update a second PR because this one was already big enough.

Reply:
Ok cool, just wanted to make sure we are not missing on that. Maybe worth to create a separate issue to track that?